hyperloglog_test.go

```diff
@@ -84,6 +85,30 @@ func TestHyperLogLogBig(t *testing.T) {
 	testHyperLogLog(t, 0, 4, 17)
 }
 
+func testReset(t *testing.T, m uint, numObjects, runs int) {
+	rand.Seed(101)
+
+	h, err := New(m)
+	if err != nil {
+		t.Fatalf("can't make New(%d): %v", m, err)
+	}
+
+	for i := 0; i < runs; i++ {
+		for j := 0; j < numObjects; j++ {
+			h.Add(rand.Uint32())
+		}
+		h.Reset()
+
```

Comment:
nit: if you want to really test it you want to verify the address to the h.Registers has not changed as well